app/viewmodels/DutyDefermentAccountsViewModel.scala

```diff
@@ -0,0 +1,317 @@
+        Some(new duty_deferment_account_direct_debit_setup()(account, finHomeModel.dutyDefermentAccountDDSetupLinks()))
+      } else {
+        None
+      }
+
+    ContentRowModel(
+      directDebitSetupComponent = directDebitSetupComponent,
+      nonDirectDebitContent =
+        if (directDebitSetupComponent.isEmpty) {
+          nonDirectDebitContent(account, finHomeModel)
+        } else {
+          None
+        }
+    )
+  }
+
+  private def nonDirectDebitContent(account: DutyDefermentAccount,
+                                    finHomeModel: FinancialsHomeModel)
+                                   (implicit messages: Messages, appConfig: AppConfig): Option[NonDirectDebitContent] = {
+
+    val balances = if (account.status != AccountStatusPending) {
+      Some(new duty_deferment_balances(new duty_deferment_balance_details)(account, finHomeModel))
+    } else {
+      None
+    }
+
+    val pendingAccountGuidance = if (account.status == AccountStatusPending) {
+      Some(new duty_deferment_account_pending().apply())
+    } else {
+      None
+    }
+
+    Some(
+      NonDirectDebitContent(
+        accountLimit = accountLimit(account),
+        balances = balances,
+        viewStatements = viewStatements(account, finHomeModel),
+        paymentDetails = paymentDetails(account, finHomeModel),
+        topUp = topUp(account),
+        pendingAccountGuidance = pendingAccountGuidance
+      ))
+  }
+
+  private def accountLimit(account: DutyDefermentAccount)
+                          (implicit messages: Messages): AccountAvailableModel = {
+
+    val positiveBalance: Option[PositiveBalanceModel] = positiveBalanceValue(account)
+
+    val negativeBalanceValue =
+      if (positiveBalance.isEmpty) {
+        Some(Formatters.formatCurrencyAmount(account.balances.availableBalance.abs))
+      } else {
+        None
+      }
+
+    AccountAvailableModel(
+      positiveBalanceValue = positiveBalance,
+      negativeBalanceValue = negativeBalanceValue)
+  }
+
+  private def viewStatements(account: DutyDefermentAccount,
+                             finHomeModel: FinancialsHomeModel)
+                            (implicit messages: Messages, appConfig: AppConfig): Option[FooterLinkModel] = {
+    if (account.status != AccountStatusPending) {
+      Some(
+        FooterLinkModel(
+          id = s"duty-deferment-account-${account.number}",
+          href = finHomeModel
+            .dutyDefermentAccountDetailsLinks()(appConfig)((account.owner, account.number): (String, String)),
+          displayValue = messages("cf.accounts.viewStatements"),
+          hiddenMsg = messages("cf.accounts.label.dan", account.number))
+      )
+    } else {
+      None
+    }
+  }
+
+  private def paymentDetails(account: DutyDefermentAccount,
+                             finHomeModel: FinancialsHomeModel)
+                            (implicit messages: Messages, appConfig: AppConfig): Option[FooterLinkModel] =
```

Comment:
for some of the private def we're encasing the def in a {} and some we are not. 
You can see here we don't as after the = there is not a {

I think we should wrap all the defs that don't have this, at the very least it makes the indictment at the end of the if statement look correct then?

Reply:
Done